makefile

```diff
@@ -0,0 +1,15 @@
+
+VOLUMES := -v $(CURDIR):/src -v $(CURDIR)/public:/target 
+IMAGE := klakegg/hugo:0.53-ext-alpine
+
```

Comment:
Maybe add `.PHONY` so we have no caching fun

Reply:
good catch, thank you - done